### ISSUE TITLE
Model for URL as object

### ISSUE THREAD

**Issue by kestlund** [OregonDigital/oregondigital#405]
Some collections have a URL as object rather than a file. (e.g. Oregon Explorer)

**Comment by jsimic** [comment]
Yes, Building Oregon is still using URLs to point to HABS, I believe. Percent for Art and Artist's Books may also have some.

**Comment by kestlund** [comment]
@jsimic: Artists' Books and Percent for Art shouldn't have any that aren't relations. What is HABS? Can you give an example link to a record?

@lsat12357 : can you separate out the links from Building Oregon into its own batch?  (Looks like there are only 6)

**Comment by jsimic** [comment]
@kestlund: Historic American Building Survey: http://www.loc.gov/pictures/collection/hh/. Found one with a State Historic Preservation Office URL in the Image Source field, but part of a string: http://contentdm.oregondigital.org/cdm4/item_viewer.php?CISOROOT=/archpnw&CISOPTR=3148&CISOBOX=1&REC=7

**Comment by kestlund** [comment]
@jsimic : that's fine. The ones we need to worry about to "model as objects" is when there is no object but just a reference to a URL.  CDM tells me there are six of those in Building Oregon.

**Comment by jsimic** [comment]
@kestlund : I think I only saw them in finding aid fields.

**Comment by jsimic** [comment]
@kestlund : tribal legacies, japanese student relocation, and all the historical photograph collections

**Comment by kestlund** [comment]
UO collections:
- 0 Building Oregon
- Oregon Latino Heritage (videos) want to add videos directly 

no files bigger than 350MB
